config/samples/simplenificluster.yaml

```diff
@@ -5,13 +5,31 @@ metadata:
 spec:
   service:
     headlessEnabled: true
-  zkAddress: "zookeeper.zookeeeper:2181"
-  zkPath: "/simplenifi"
-  clusterImage: "apache/nifi:1.15.3"
-  oneNifiNodePerNode: false
+    labels:
+      cluster-name: simplenifi
+  zkAddress: "zookeeper.zookeeper:2181"
+  zkPath: /simplenifi
+  externalServices:
```

Comment:
IMO, should probably retain all of the example `externalServices` as sample of what they each look like.

Reply:
Kind succeed to make the cluster work with all example of externalServices. What is the purpose of exposing this, every K8S user should  already be aware of service type?

Reply:
I do agree with that. I don't think it's strictly necessary, but it just serves as an example of what each service type looks like in a `NifiCluster` spec.

Reply:
For the simplenificluster we can keep it as short as possible, the details should be in the https://github.com/konpyutaika/nifikop/blob/master/config/samples/nifi_v1alpha1_nificluster.yaml file